src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -179,14 +180,19 @@ public static boolean isWater(final IBlockState iBlockState)
     /**
      * Checks if a certain block returns a seed as the item.
      *
+     * Suppressing Sonar Rule squid:S2259
+     * This rule does "Null pointers should not be dereferenced"
+     * But in this case the rule does not apply because
+     * ItemStackUtils.isEmpty return true is stack is null
      * @param world the world the block is in.
      * @param pos   the position the block is at.
      * @return true if is a seed.
      */
+    @SuppressWarnings("squid:S2259")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S2259' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)